packages/apollo-link/src/index.ts

```diff
@@ -0,0 +1,64 @@
+import * as apolloImport from '@apollo/client'
+import {
+  LoadFromUrlOptions,
+  SubscriptionProtocol,
+  UrlLoader,
+} from '@graphql-tools/url-loader'
+import { type ExecutionRequest, isAsyncIterable } from '@graphql-tools/utils'
```

Comment:
I recall some PR where @n1ru4l tried this new syntax and we had to revert 🤔 so maybe we should avoid doing this

Reply:
Ok I removed those 👍